fe/fe-core/src/main/java/com/starrocks/catalog/TableFunctionTable.java

```diff
@@ -142,19 +153,114 @@ public TableFunctionTable(Map<String, String> properties) throws DdlException {
     }
 
     // Ctor for unload data via table function
-    public TableFunctionTable(String path, String format, String compressionType, List<Column> columns,
-            @Nullable List<Integer> partitionColumnIDs, boolean writeSingleFile, long targetMaxFileSize,
-            Map<String, String> properties) {
+    public TableFunctionTable(List<Column> columns, Map<String, String> properties, SessionVariable sessionVariable) {
```

Comment:
Do nt throw exeption in constructor， put these analyze code in a function

Reply:
The other constructor used by `SELECT * FROM FILES` could also throw exceptions. So it is not quite easy to change.